src/tensor_products.jl

```diff
@@ -251,24 +252,8 @@ end
     return Tensor{2, dim}(tomatrix(S1_t) * tomatrix(S2_t))
 end
 
-# Specialized methods for symmetric tensors
-@generated function Base.dot{dim}(S1::SymmetricTensor{2, dim}, v2::Vec{dim})
```

Comment:
This was not even specialized. I still think its weird that its faster to convert to a `Tensor` and then use their `dot`...